python/lsst/ctrl/bps/wms/panda/conf_example/pipelines_check_idf.yaml

```diff
@@ -1,13 +1,17 @@
-pipelineYaml: "${OBS_SUBARU_DIR}/pipelines/DRP.yaml#processCcd"
+includeConfigs:
+- ${CTRL_BPS_DIR}/config/bps_idf.yaml
+
+operator: ${USER}
```

Comment:
Double check whether this is needed.  If operator isn't set in the submit yaml, bps should grab the value of that environment variable right after reading the yaml file.

Reply:
The default sounds good.   I removed this line.